utilities/virt.py

```
@@ -2111,22 +2112,59 @@ def vm_instance_from_template(
+            f"nohup oc adm drain {node.name} --delete-emptydir-data --ignore-daemonsets=true --force &",
             shell=True,
         )
         yield
     finally:
-        if node_mgmt == "drain":
-            LOGGER.info("Terminate drain process")
-            run(
-                shlex.split('pkill -f "oc adm drain"'),
-            )
+        LOGGER.info("Terminate drain process")
+        run(shlex.split('pkill -f "oc adm drain"'))
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)